src/modules/rabbitmq/blockchain-events/nft-events.service.ts

```diff
@@ -71,8 +72,23 @@ export class NftEventsService {
               }
             }
           }
+          break;
 
+        case NftEventEnum.ESDTModifyCreator:
+        case NftEventEnum.ESDTMetaDataRecreate:
+        case NftEventEnum.ESDTMetaDataUpdate:
+        case NftEventEnum.ESDTNFTUpdateAttributes:
+          const updateEvent = new UpdateAttributesEvent(event);
+          const updateTopics = updateEvent.getTopics();
+          const collectionUpdateInfo = await this.mxApiService.getCollectionByIdentifierForQuery(updateTopics.collection, 'fields=name,type');
+          if (collectionUpdateInfo?.type === NftTypeEnum.NonFungibleESDT || collectionUpdateInfo?.type === NftTypeEnum.SemiFungibleESDT) {
+            await this.triggerCacheInvalidation(
+              `${updateTopics.collection}-${updateTopics.nonce}`,
```

Comment:
Double check: nonce is here as hex (not number), correct?

Reply:
yes